blockchain/ethereum/internal/chain_test.go

```diff
@@ -60,16 +60,16 @@ func Test_ChainBackend_ValidateContracts(t *testing.T) {
 func Test_ChainBackend_NewFunder(t *testing.T) {
 	rng := rand.New(rand.NewSource(1729))
 	setup := ethereumtest.NewChainBackendSetup(t, rng, 1)
-	randomAddr1 := ethereumtest.NewRandomAddress(rng)
+	assetAddr := setup.AssetAddr
+	addr := setup.Accs[0].Address()
 
-	assert.NotNil(t, setup.ChainBackend.NewFunder(randomAddr1))
+	assert.NotNil(t, setup.ChainBackend.NewFunder(assetAddr, addr))
```

Comment:
Only requirement here is the parameters should be any valid address, need not be onChain or adjAddr. So use randomAddr as before. 
```suggestion
	assert.NotNil(t, setup.ChainBackend.NewFunder(randomAddr1, randomAddr2))
```

Reply:
[Done](https://github.com/hyperledger-labs/perun-node/pull/114/commits/aebcf94494c5d13ad272ecadd28edfe7b22dc659).